src/WebJobs.Extensions.DurableTask/AzureStorageDurabilityProviderFactory.cs

```diff
@@ -41,7 +41,7 @@ internal class AzureStorageDurabilityProviderFactory : IDurabilityProviderFactor
             this.defaultConnectionName = this.azureStorageOptions.ConnectionStringName ?? ConnectionStringNames.Storage;
         }
 
-        public string Name => "Azure Storage";
+        public string Name => "AzureStorage";
```

Comment:
To avoid duplication, I wonder if either this property should reference DurableTaskExtension.DefaultProvider or (perhaps slightly better) you define an internal const in this class and have both the `Name` property here and the DurableTaskExtension code both reference that const.